projectleader_mailinglist.txt

```diff
@@ -0,0 +1 @@
+aamer@anl.gov, gabriel.antoniu@inria.fr, balaji@anl.gov, gbauer@ncsa.illinois.edu, leonardo.bautista@bsc.es, Anne.Benoit@ens-lyon.fr, cappello@anl.gov, emconsta@mcs.anl.gov, mdorier@anl.gov, guillaume.houzeaux@bsc.es, hovland@mcs.anl.gov,   email:, imamura.toshiyuki@riken.jp, emmanuel.jeannot@inria.fr, i.kabadshow@fz-juelich.de, keahey@mcs.anl.gov, ktesser@inf.ufrgs.br, t.klatt@fz-juelich.de, A.Lintermann@fz-juelich.de, lukeo@illinois.edu,  antonio.pena@bsc.es, tpeterka@anl.gov, d.pleiter@fz-juelich.de, ramet@labri.fr, rross@mcs.anl.gov, schleife@illinois.edu, omer.subasi@bsc.es, b.wylie@fz-juelich.de, 
```

Comment:
Uh! There is a `email: ` in the list.. probably from someone who did not specify an email address. You need to catch this case (and maybe through a warning like 'no email address found for ...').